src/strings/global/en-US.json

```diff
@@ -0,0 +1,144 @@
+                "control" : {
+                    "lang-selector" : {
+                        "tooltip" : "Choose language",
+                        "null-choice" : "Choose language"
+                    },
+                    "upload-button-tooltip" : "Load a file into the editor. Overwrite existing content.",
+                    "submit-button" : "Submit"
+                }
+            },
+            "notify" : {
+                "success" : "Successfully submitted solution",
+                "error" : {
+                    "fail-stat" : "Failure during submission : {error}",
+                    "fail-trans" : "Failed to submit. It seems like a transmission error..."
+                }
+            }
+        },
+        "submissions" : {
+            "launcher" : "Submissions",
+            "control" : {
+                "reload-button" : {
+                    "reload" : "Reload",
+                    "reloading" : "Updating...",
+                    "tooltip" : "Update the data & change back to first page"
+                },
+                "rows-per-page-change" : "Rows per page : ",
+                "index-of-total" : "{from}-{to} of {count}"
+            },
+            "table" : {
+                "contestant" : "Submitted by",
+                "problem" : "Problem",
+                "language" : "Programming language",
+                "verdict" : "Verdict",
+                "executionTime" : "Execution duration",
+                "memory" : "Memory consumed",
+                "timestamp" : "Timestamp",
+                "points" : "Points awarded"
+            },
+            "details" : {
+                "control" : {
+                    "show-source" : "View source code",
+                    "close": "Close"
+                },
+                "table" : {
+                    "no-detail" : [
+                        "No test-based judgement details available.",
+                        "I guess this is an ACM problem?"
+                    ]
+                },
+                "code" : {
+                    "panel" : {
+                        "loading" : "Loading source code...",
+                        "no-code" : [
+                            "No code is available.",
+                            "What are you expecting?"
+                        ]
+                    },
+                    "dialog.controls.close" : "Close"
+                }
+            },
+            "verdict" : {
+                "AC" : "Accepted",
+                "WA" : "Wrong output",
+                "TLE" : "Execution timed out",
+                "MLE" : "Memory limit exceeded",
+                "RTE" : "Runtime error",
+                "CE" : "Compilation error",
+                "Pending" : "Pending"
+            },
+            "error" : {
+                "submissions" : "Failed to load submissions' info.",
+                "source" : "Error occurred while loading source code."
+            }
+        }
+    }
+}
```

Comment:
Can you standardize the resource name ? It cannot be both pascal case and dash `-`

Reply:
Addressed with d6e677a0525911f821416605312c5994520c69da.